init/systemd/00-network.link

```diff
@@ -0,0 +1,6 @@
+[Match]
+OriginalName=*
+
+[Link]
+NamePolicy=kernel
```

Comment:
The name policy here will keep the kernel name because there is only one network ethernet interface, so it is consistent.

Reply:
What does this `NamePolicy=kernel` actually means?

Reply:
@danko-miladinovic would a super-simple file like this work:
```bash
[Link]
Name=eth0
```

Do we need part
```bash
[Match]
OriginalName=*
```

Reply:
Kernel can give other names - like enp0s3` for example.

I think it would be safer to use:
```bash
[Match]
OriginalName=*

[Link]
Name=eth0
```

but why are we doing all of this? We should fetch this info (which iface to use) from Qemu command line, right?